### PR TITLE
[fix/improvement]fixed the MYSQL condition issue #341 add other checks to the installer 

### PR DESCRIPTION
**Pull request by eshandhawan51**
Please prefix your pull request with one of the following: **[FEATURE]** **[FIX]** **[IMPROVEMENT]**.

**In raising this pull request, I confirm the following (please check boxes):**

- [x] I have read and understood the [contributors guide](https://github.com/CCExtractor/sample-platform/blob/master/.github/CONTRIBUTING.md).
- [x] I have checked that another pull request for this purpose does not exist.
- [x] I have considered, and confirmed that this submission will be valuable to others.
- [x] I accept that this submission may not be used, and the pull request closed at the will of the maintainer.
- [x] I give this submission freely, and claim no ownership to its content.

**My familiarity with the project is as follows (check one):**

- [ ] I have never used the project.
- [x] I have used the project briefly.
- [ ] I have used the project extensively, but have not contributed previously.
- [ ] I am an active contributor to the project.

---

fixed issue #341 and added conditions for not to leave admin email and password empty also removed the "not a valid identifier" warning

### CODE REVIEW

File: install/install.sh [hunk 1]
```diff
@@ -40,6 +40,9 @@ do
                 if [ "$mysql_user_resp" = "Q" ]; then
                     exit 1
                 fi
+            fi
+			if [  -f "$file" ]; then
```
Comment (canihavesomecoffee) [inline install/install.sh]: Your alignment is not ok. Please correct.

File: install/install.sh [hunk 2]
```diff
@@ -55,7 +58,7 @@ echo "-------------------------------"
 echo ""
 echo "In order to configure the platform, we need some information from you. Please reply to the following questions:"
 echo ""
-read -e -r -p -s "Password of the 'root' user of MySQL: " -i "" db_root_password
+read -e -r -p  "Password of the 'root' user of MySQL: " -i "" db_root_password
```
Comment (canihavesomecoffee) [inline install/install.sh]: Why remove the -s? It's needed to not echo the input back, which is on purpose in this case.

File: install/install.sh [hunk 3]
```diff
@@ -154,12 +157,24 @@ echo ""
...
+   echo "Please Re-enter password again ... "
+   read -e -r -p  "Admin password: " admin_password
+   read -e -r -p  "Confirm admin password: " confirm_admin_password
+
+done 
+while [ $admin_password != $confirm_admin_password ]; do
     echo "Entered passwords did not match! Retrying..."
-    read -e -r -p -s "Admin password: " admin_password
-    read -e -r -p -s "Confirm admin password: " confirm_admin_password
+    read -e -r -p "Admin password: " admin_password
```
Comment (canihavesomecoffee) [inline install/install.sh]: Same question here.

File: install/install.sh (hunk 2 above)
Comment (eshandhawan51) [reply install/install.sh]: -s is the reason for the "not a valid identifier" 
 read: `Password of the 'root' user of MySQL: ': not a valid identifier
which in turn blocks the input

File: install/install.sh (hunk 1 above)
Comment (eshandhawan51) [reply install/install.sh]: I will create a new PR for the alignment

File: install/install.sh (hunk 3 above)
Comment (eshandhawan51) [reply install/install.sh]: it has a much worse effect here as it ends up creating a loop of your passwords do not match at this place and blocks the input

File: install/install.sh (hunk 2 above)
Comment (eshandhawan51) [reply install/install.sh]: sending a new PR with corrections

**Pull request closed without merging**